src/utilities/bcl/LocalBCL.cpp

```diff
@@ -167,17 +169,29 @@ namespace openstudio{
 
   void LocalBCL::close()
   {
+    // Close the connection to the database if needed.
+    // TODO: something is bugging me here. why are we using a std::shared_ptr and not unique_ptr?
+    // shared_ptr -> reset() should destroy the object (which calls closeConnection) if it's only reference left, which it should...
+    // TODO: probabably not needed
+    instanceInternal()->closeConnection();
+    // reset pointer
```

Comment:
TODO?

Reply:
@kbenne @tijcolem review please?

Reply:
is the caller managing the lifetime of the db connection? I would think so. In that case, I think it should be changed to a `std:unique_ptr` point and not shared.  As you indicated ptr.reset() should delete the obj and therefore call the destructor. Regardless, the way you have it now by explicitly closing the connection is good for code readability anyway.